src/core/Form/Select/MultiSelect/MultiSelectChipList/MultiSelectChipList.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { RefObject } from 'react';
+import { ChipList, Chip } from '../../../../Chip';
+
+import { MultiSelectData } from '../MultiSelect/MultiSelect';
+
+export interface MultiSelectChipListProps<T extends MultiSelectData> {
+  selectedItems: (T & MultiSelectData)[];
+  ariaChipActionLabel?: string;
+  sourceRef?: RefObject<any>;
+  onClick?: (item: T & MultiSelectData) => void;
```

Comment:
Having JSDoc for these might benefit us later.

Reply:
Added for the necessary parts.